packages/server/graphql/public/mutations/addFeatureFlagToOrg.ts

```diff
@@ -0,0 +1,34 @@
+import getRethink from '../../../database/rethinkDriver'
+import {getUserId, isSuperUser} from '../../../utils/authorization'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const addFeatureFlagToOrg: MutationResolvers['addFeatureFlagToOrg'] = async (
+  _source,
+  {orgIds, flag},
+  {authToken}
+) => {
+  const r = await getRethink()
+
+  // AUTH
+  const viewerId = getUserId(authToken)
+
+  if (!isSuperUser(authToken)) {
+    return standardError(new Error('Not authorised to add feature flag'), {
+      userId: viewerId
+    })
+  }
+
+  // RESOLUTION
+  await r
+    .table('Organization')
+    .getAll(r.args(orgIds), {index: 'id'})
+    .update({
+      featureFlags: r.row('featureFlags').default([]).setInsert(flag)
+    })
+    .run()
+
+  return {orgIds}
```

Comment:
-1 I think we should return the updated `orgIds` from the query above rather than returning the unchanged `orgIds` input. Outputting the input doesn't tell us much, but returning the updated orgIds tells us which orgs now have the feature flag

Reply:
@nickoferrall When I do the update operation I can specify fields to return. In this particular case I would see the resulting `teamsLimit` value for each org, I can also query `updatedAt` field if needed.

If I return only actually updated organizations, in some cases there may be some orgs missing in the answer and I will not know why, either I made a mistake in the id, or nothing has changed.

What do you think?

Reply:
I think it could be confusing if the Parabol employee runs the query with several orgIds, one of them fails to update, but they see that organization in the results without any error message. If they don't query `teamsLimit` and just query the org `name`, for example, they would assume the org had updated.

Also, even if the Parabol employee does query the `teamsLimit`, but they're updating 100 org ids, it would be a pain reading through all of the responses to see if one failed.

Perhaps we could return `updatedOrgIds` and `notUpdatedOrgIds`? Like [this example](https://github.com/ParabolInc/parabol/blob/13cabc3c6bd6b2082c2d7e37a46695a1498f6221/packages/server/graphql/private/types/ChangeEmailDomainSuccess.ts)

Reply:
Make sense, made `updatedOrganizations` instead of `organizations`